tests/run-make/jobserver-error/rmake.rs

```diff
@@ -0,0 +1,55 @@
+//! If the environment variables contain an invalid `jobserver-auth`, this used to cause an ICE
+//! until this was fixed in [do not panic on failure to acquire jobserver token
+//! #109694](https://github.com/rust-lang/rust/pull/109694).
+//!
+//! Proper handling has been added, and this test checks that helpful warnings and errors are
+//! printed instead in case of a wrong jobserver. See
+//! <https://github.com/rust-lang/rust/issues/46981>.
+
+//@ only-linux
+//@ ignore-cross-compile
+
+use run_make_support::{diff, rustc};
+
+fn main() {
+    let out = rustc()
+        .stdin_buf(("fn main() {}").as_bytes())
+        .env("MAKEFLAGS", "--jobserver-auth=5,5")
```

Comment:
```suggestion
        .env("MAKEFLAGS", "--jobserver-auth=1000,1000")
```
use some higher FDs to guard against rustc using more fds initially

Reply:
Switched to higher FDs.